kymatio/scattering2d/backend/torch_backend.py

```diff
@@ -0,0 +1,328 @@
+# Authors: Edouard Oyallon, Sergey Zagoruyko
```

Comment:
I think it would be best if the documentation here was a model for how the documentation should look in the rest of the project. 

An example is that some of the documentation seems to mention the fact that the input has two dimensions. Do we want this? Do we want to more verbose in this regard?

Reply:
@janden @eickenberg ?

Reply:
That's a good question actually!

Reply:
I'd say with a point?

Reply:
I'm OK with it.

Reply:
Ok. I think @MuawizChaudhary addressed most of these already.